commit/chainfee/outcome.go

```diff
@@ -20,22 +20,28 @@ import (
 
 func (p *processor) Outcome(
 	ctx context.Context,
-	_ Outcome,
+	prevOutcome Outcome,
 	_ Query,
 	aos []plugincommon.AttributedObservation[Observation],
 ) (Outcome, error) {
 	lggr := logutil.WithContextValues(ctx, p.lggr)
 
+	inflightPricesOutcome := newInflightPricesOutcome(
+		prevOutcome.InflightChainFeeUpdates, prevOutcome.InflightRemainingChecks-1)
+
 	consensusObs, err := p.getConsensusObservation(lggr, aos)
 	if err != nil {
-		return Outcome{}, fmt.Errorf("get consensus observation: %w", err)
+		return inflightPricesOutcome, fmt.Errorf("get consensus observation: %w", err)
```

Comment:
Wouldn't this be better left as empty? It should not get used anyway, if an error is returned.

Reply:
This is an error in building the consensus observation, could that be transient? seems like it shouldn't be